src/contrib/mlir/core/compiler.cpp

```diff
@@ -60,6 +60,10 @@
 // Defines a new LLVM debug type for this file to be used by LLVM_DEBUG macro.
 #define DEBUG_TYPE "mlir-compiler"
 
+static llvm::cl::opt<bool> clEnableNgOpFusion("ngraph-op-fusion",
```

Comment:
I'm adding exactly the same flag for the generic op fusion pass :). Could we rename this to be more specific? Something like `ngraph-mkldnn-op-fusion` or `ngraph-kernel-lib-fusion` or similar...